### PR TITLE
[Behat] No more weird fatal errors calling getMasterRequest() on null

### PR DESCRIPTION
**Pull request by pamil**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: app/TestKernel.php [hunk 1]
```diff
@@ -65,6 +66,6 @@ protected function cleanupContainer(ContainerInterface $container)
 
     protected function getContainerBaseClass()
     {
-        return '\PSS\SymfonyMockerContainer\DependencyInjection\MockerContainer';
+        return MockerContainer::class;
```
Comment (pamil) [inline app/TestKernel.php]: I had to do it :)

**Comment by pamil** [comment]
Let's extra check the memory usage after this changes.

**Comment by pamil** [comment]
Looks fine! :)

**Comment by michalmarcinkowski** [comment]
Awesome! :+1:

**Comment by pjedrzejewski** [comment]
Thanks Kamil!

**Pull request merged**